More css fixes

Gwenwyfar opened the pull request.

Gwenwyfar committed:
Rework inputs and buttons padding/height

This makes their height scalable with font-size while keeping it consistent.

Fixes #4637

Signed-off-by: Gwenwyfar <gwenwyfar@simplemachines.org>

Gwenwyfar committed:
Minor css adjustments

Signed-off-by: Gwenwyfar <gwenwyfar@simplemachines.org>

Gwenwyfar committed:
Remove unused classes

These no longer serve any purpose.

Signed-off-by: Gwenwyfar <gwenwyfar@simplemachines.org>

Gwenwyfar committed:
Fixes gaps between title bars and content

Signed-off-by: Gwenwyfar <gwenwyfar@simplemachines.org>

Gwenwyfar committed:
General css code cleanup

Unused rules and classes, rearranging rules, etc.

Signed-off-by: Gwenwyfar <gwenwyfar@simplemachines.org>

Gwenwyfar committed:
Fixes topic titles being cut and lines overlapping

Signed-off-by: Gwenwyfar <gwenwyfar@simplemachines.org>

XinYenFon commented on Themes/default/css/index.css:
```diff
@@ -3522,6 +3487,11 @@ h3.catbg .generic_icons::before, h3.catbg .icon {
 	vertical-align: middle;
 	margin: 0 5px 0 0;
 }
+.cat_bar + .windowbg2, .cat_bar + .windowbg, .cat_bar + .roundframe {
```
I'm just curious related to this and missing .information + .windowbg/2 rest is nice ^^

Gwenwyfar committed:
Bye bye windowbg2

Converts all windowbg2 classes into windowbg. There's no longer any difference between them, so it is just redundancy.

Signed-off-by: Gwenwyfar <gwenwyfar@simplemachines.org>

XinYenFon commented on Themes/default/Calendar.template.php:
```diff
@@ -1025,13 +1025,13 @@ function template_hms()
 	{
 		echo '
-			<tr class="', $alt ? 'windowbg2' : 'windowbg', '">
+			<tr class="', $alt ? 'windowbg' : 'windowbg', '">
```
:P

Gwenwyfar committed:
Replace windowbg conditionals

Signed-off-by: Gwenwyfar <gwenwyfar@simplemachines.org>

XinYenFon reviewed (changes requested):
You also need to clean up the $alt definitions from removed functions.

Gwenwyfar commented:
You're aware that entire section is the clock easter egg right?

XinYenFon commented:
Not really :P I just checked the code 😆 not the func.

jdarwood007 commented:
Quick testing shows nothing I see as breaking.  Antes gave a thumbs up as well
